homeassistant/components/switch/rest.py

```diff
@@ -69,8 +69,9 @@ def is_on(self):
 
     def turn_on(self, **kwargs):
         """Turn the device on."""
+        headers = {'content-type': 'application/x-www-form-urlencoded'}
```

Comment:
Yes, that's a great idea, however I don't know how to do it, so please share your code if you have a chance.

Thank you!